2-mandatory/3-bush-berries.js

```diff
@@ -21,9 +21,52 @@
   Let's first look at an example that will teach you how to use these methods.
 */
 
-function isBushSafe(berryArray) {
-  //Write your code here
-}
+function isBerrySafe(berry){
+    if(berry=== "pink"){
```

Comment:
If you download prettier for JS in your VS code, it makes your codes look nicer! no extra space, etc